### ISSUE TITLE
Multiple password popups in Linux

### ISSUE THREAD

**Issue by roelofz** [allotropia/nextcloud_files_libreoffice_edit#30]
Hi,

How does this work?
I installed the app in Nextcloud, I can see the pull down menu: open with, then a lot of password prompts appear (with and without user name, with username Nextcloud username applied and password) resulting in not opening the file.
So where did I go wrong?

**Comment by roelofz** [comment]
This app is no longer maintained?

**Comment by alkanex-xyz** [comment]
It works fine... In my case the prompts are created from libreoffice. It needs credentials to read the file...

The first one ist for the masterpassword of the libreoffice password storage. The second one is for the credentials to login to nextcloud if you didn't provide a masterpassword

**Comment by roelofz** [comment]
Good you are looking with me into this.
I think the requirements are missing in the description of the app.
I get multiple prompts for my Nextcloud password (server dns name is mentioned in the popup), being sure using the right username/password.

This may be related to 2Fa or to network (webdav) rquirements or something else.
So this needs more directions in order to make it usable for a broader audience.

**Comment by alkanex-xyz** [comment]
I'm not a dev or contributor. Write some explanation for the parts that are unclear in your opinion and ask @smehrbrodt to add them in the store.

**Comment by roelofz** [comment]
I assumed you are not the dev, still cool you answer.
If I knew the requirements, I would add them, so I think it starts there. 
What to do next after installing nextcloud and installing this app?
The client requirements are described, but it looks there are also server requirements, as my setup is not working on a local network. I am using AIO, so the docker version of Necxtcloud (fool proof install of all features).

**Comment by roelofz** [comment]
Is the maintainer of this app still available?

**Comment by smehrbrodt** [comment]
Yes. As you can see in the [source](https://github.com/allotropia/nextcloud_files_libreoffice_edit/blob/master/src/files-action.js#L53), all that this app does is to redirect to `vnd.libreoffice.command:ofe|u|<nextcloud_webdav_url>`.

Everything else is not part of this app. Password popups are from LibreOffice, so issues need to be filed there if necessary. Feel free to improve documentation though through a PR.

**Comment by roelofz** [comment]
I see the code, but...  I can not determine if some code is missing, causing this or this is an issue of libreoffice. 
I would help in improving the documentation if I knew how. 
it looks like this feature is seldom used in combination with 2FA, at least I can not find any info on it.
So it looks like a needle in a haystack getting this too work.

If anyone knows how to set this right, I will be happy to document it in a PR.
